packages/SwingSet/src/kernel/vatManager/deliver.js

```diff
@@ -81,20 +81,23 @@ export function makeDeliver(tools, dispatch) {
     );
   }
 
-  async function deliverOneNotification(primaryVpid, resolutions) {
-    const errmsg = `vat[${vatID}].promise[${primaryVpid}] failed`;
-    return doProcess(['notify', primaryVpid, resolutions], errmsg);
+  async function deliverOneNotification(resolutions) {
+    const errmsg = `vat[${vatID}].notify failed`;
+    return doProcess(['notify', resolutions], errmsg);
   }
 
   // vatDeliverObject is:
   //  ['message', target, msg]
   //   target is vid
   //   msg is: { method, args (capdata), result }
-  //  ['notify', vpid, resolutions]
-  //   vpid is the id of the primary promise being resolved
-  //   resolutions is an object mapping vpid's to the final promise data,
-  //     rendered in vat form, for both the primary promise and any collateral
-  //     promises it references whose resolution has been newly discovered
+  //  ['notify', resolutions]
+  //   resolutions is an array of pairs: [vpid, resolution]
+  //    vpid is the id of the primary promise being resolved
```

Comment:
I think the word "primary" in this comment is vestigal. But it might be worth noting that the first element of the array is the primary/initial promise, and that the others are ordered to ensure that no promise is resolved before it is first introduced.